index.html

```diff
@@ -410,7 +415,7 @@ <h3>What should the .csv file Look Like?</h3>
                 reader.onload = function(e) {
                     csvResult = e.target.result;
                     csvResult = $.csv.toObjects(csvResult);
-                    keys = identifyCSVKeys(csvResult);
+                    keys = identifyCSVKeys(csvResult, dose);
```

Comment:
IS there any reason this change is needed for the zebra audio pull?

Reply:
You're right, that shouldn't be there.  The correct line is in the hybrid fix, ie that line should say:

` keys = identifyCSVKeys(csvResult, doseNumber);
`

Reply:
Actually, for this change in isolation the line would be keys = identifyCSVKeys(csvResult)

Do I make that change and resubmit or do you do that by not accepting the change to that line?

Reply:
I think I've committed a change that resolves this.  Fingers crossed ...